gf/include/alps/gf/grid.hpp

```diff
@@ -69,11 +69,12 @@ namespace alps {
           double scale = std::log(t_max_ / t_min_) / ((float) ((nfreq_ / 2 - 1)));
           points[nfreq_ / 2] = c_;
           for (int i = 0; i < nfreq_ / 2; ++i) {
-            if(i<nfreq_/2)
+            /// check boundaries for an even # of frequencies
+            if(i<nfreq_/2 - 1)
               points[nfreq_ / 2 + i + 1] = c_ + t_min_ * std::exp(((float) (i)) * scale);
             points[nfreq_ / 2 - i - 1] = c_ - t_min_ * std::exp(((float) (i)) * scale);
           }
-          //if we have an odd # of frequencies, this catches the last element
+          ///if we have an odd # of frequencies, this catches the last element
```

Comment:
Is it intentionally a documenting (Doxygen-parsed) comment?
It looks a bit odd in the generated doc, so i'll make it just `//` for now.